config/routes.rb

```diff
@@ -5,5 +5,5 @@
     get :payment_confirmation, controller: :intents
     get :setup_confirmation, controller: :intents
   end
-  resources :webhooks, only: :create, format: false
+  post '/webhooks/:suffix', format: false, to: 'webhooks#create'
```

Comment:
I was wondering if it made sense to scope everything within a sort if payment method id/code that identifies it without being the sequential number, thoughts on this?

Reply:
Not sure I get the point. Do you mind using the `suffix` also to scope the `get :payment_confirmation` & `get :setup_confirmation` routes?

Reply:
Yes I meant that, once we have a code identifying the payment method is more consistent to scope all the routes in the same way

Reply:
I see your point. However, on the other side, it also makes sense to stick to the Rails default way unless there's a compelling reason, and that reason only exists for the webhook endpoint. Regardless, are you ok if we discuss that on a separate ticket? Ultimately, that's outside of the scope of this one's purpose.

Reply:
Sure thing, it's not blocking 👍

Reply:
Cool! I created https://github.com/solidusio/solidus_stripe/issues/220. Thanks, @elia!